data-tool/flows/tombstone/tombstone_mappings.py

```diff
@@ -308,11 +309,8 @@ def has_value(cls, value):
     EventFilings.FILE_AM_RM: 'Amendment - Receiver or Receiver Manager',
     EventFilings.FILE_AM_SS: 'Amendment - Share Structure',
 
-    # TODO: Amalgamation Out Consent - unsupported
-    # IAMGO: 'Application For Authorization For Amalgamation (into a Foreign Corporation) with 6 months consent granted'
-    # TODO: Amalgamation Out - unsupported
-    # AMALO: 'Record of Amalgamation'
-
+    EventFilings.FILE_IAMGO: '6-Month Consent to Amalgamate Out',
```

Comment:
The display name is the longer one - `Application For Authorization For Amalgamation (into a Foreign Corporation) with 6 months consent granted`.

Reply:
Is that required for the migration filings? For the new one it's not https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/core/meta/filing.py#L339

Reply:
Yes, we need to display the same name in COLIN no matter what name is using in new system.